mcp_server/src/resources/resource_handlers.ts

```
@@ -0,0 +1,394 @@
+    //         },
+    //       },
+    //     ],
+    //   };
+    // }
+    try {
+      switch (parsedUri.type) {
+        case "root":
+          const rootResult = await rpcToolHandlers.handleToolRequest(
+            "inspector_get_root_widget",
+            {}
+          );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)